randomizer.py

```diff
@@ -1152,6 +1156,169 @@ def mutate(self):
+            if value > 0 and randint(1,10) != 1:
+                # 1% Chance for a pre-existing Inflict Status to be randomized; 10% otherwise
+                return
+            newvalue = randint(0,0x79)
+            if newvalue == 0x60:
+                # Banning Crystal (since it's more likely to appear on weapons)
+                return
+            self.inflict_status = newvalue
+
+
+class ShieldObject(TableObject):
+    def mutate(self):
+        for attr in ["physical_evade", "magic_evade"]:
+            value = getattr(self, attr)
+            if random.choice([True, False]) and 0 <= value <= 0x50:
+                    newvalue = mutate_normal(value, minimum=0, maximum=0x50) # Max 80
+                    setattr(self, attr, newvalue)
+
+
+class ArmorObject(TableObject):
+    def mutate(self):
+        for attr in ["hp_bonus", "mp_bonus"]:
+            value = getattr(self, attr)
+            if random.choice([True, False]) and 0 <= value <= 0xFD:
+                    newvalue = mutate_normal(value, minimum=0, maximum=0xFD)
+                    setattr(self, attr, newvalue)
+
+
+class AccessoryObject(TableObject):
+    def mutate(self):
+        for attr in ["physical_evade", "magic_evade"]:
+            value = getattr(self, attr)
+            if random.choice([True, False]) and 0 <= value <= 0x3C:
+                    newvalue = mutate_normal(value, minimum=0, maximum=0x3C) # Max 60
+                    setattr(self, attr, newvalue)
+
+
+class ChemistItemObject(TableObject):
+    def mutate(self):
+        value = self.zval
+        if 1 <= value <= 0xFD:
+            newvalue = mutate_normal(value, minimum=1, maximum=0xFD)
+            self.zval = newvalue
+
+
+class InflictStatusObject(TableObject):
+    def randomize_empty(self):
+        if (0x1D <= self.index <= 0x1F) or  (0x7A <= self.index <= 0x7F):
+            toinflict = 0
+            while True:
+                toinflict |= (1 << randint(0, 39))
+                toinflict &= VALID_START_STATUSES
+                if (toinflict and
+                        randint(1, 2**(bin(toinflict).count("1"))) != 1):
+                    break
+            self.statuses_to_inflict = toinflict
+            if not (self.statuses_to_inflict == 0x0000000000):
+                choice = randint(1,9)
+                if choice <= 3: # 33%
+                    self.set_bit("random", True)
+                elif choice <= 6: # 33%
+                    self.set_bit("separate", True)
+                elif choice <= 8: # 22%
+                    self.set_bit("cancel", True)
+                else: # 11%
+                    self.set_bit("all_or_nothing", True)
+
+
+class ItemAttributesObject(TableObject):
+    def mutate(self):
+        if self.index > 0:
+            if self.index <= 49:
+                for attr in ["pa", "ma", "speed", "move", "jump"]:
+                    value = getattr(self, attr)
+                    if random.choice([True, False]) and 0 <= value <= 0xFD:
```

Comment:
I slightly increased the odds of triggering this randomization (from 1/3 to 1/2). My reasoning is that mutate_normal already resists extreme randomizations, so a value of zero will _very rarely_ increase in the first place.